src/services/babel-browser.gen.js

```diff
@@ -0,0 +1,68 @@
+const babel = require('@babel/core');
+const presetEnv = require('@babel/preset-env');
+
+const browserList = require('../../config/browsers.json');
+
+module.exports = () => {
+  const browserListForBabel = {};
+
+  for (const browser in browserList) {
+    if (browser !== 'chromium') {
+      browserListForBabel[browser] = browserList[browser];
+    }
+  }
+
+  // preset-env doesn't give an introspection API, so we have to get it from
+  // the metadata on transformed code
+  const result = babel.transformSync('', {
+    sourceType: 'script',
+    presets: [
+      [
+        presetEnv,
+        {
+          targets: browserListForBabel,
+          // The module for regenerator causes an import error, so we ignore it
+          exclude: ['transform-regenerator'],
+        },
+      ],
+    ],
+  });
+
+  const keys = result.options.plugins.map(
+    plugin => plugin.key,
+  ).filter(
+    key => key.indexOf('/') !== 0,
+  );
+
+  const plugins = keys.map(key => `require("@babel/plugin-${key}")`).
```

Comment:
I suppose it’s ultimately neither here nor there, but if we fully lean into the premise that a `.gen.js` file is emitting a `code` property that stands in for actual source code we would put in a typical `.js` file, wouldn’t we want to do static `import`s here?